testing/tests/DevExpress.core/component.tests.js

```diff
@@ -1442,12 +1442,12 @@ QUnit.module("event API", {
         component.on("event", h1);
         component.on("event", h2);
 
-        component.fireEvent("event");
+        component._eventsStrategy.fireEvent("event");
```

Comment:
Should we change the previous classes' API? I mean, we don't change tests exclude events files.

Reply:
The `fireEvent` is private method, but in some tests it is checking.

Reply:
We decided to remove it from classes and call inside. Should we back it?

Reply:
I think it is ok to get rid of `fireEvent` method. But events mixin is a public class. Is it braking change? @NickMitrokhin

Reply:
The fireEvent method is not public, but some users use it. For example, https://supportcenter.devexpress.com/ticket/details/T656964/customstore-how-to-trigger-the-modified-event

Reply:
It is not legal

Reply:
This is an undocumented method. So, we can change it according to our needs without any notifications.